app/components/Comments/CommentView.tsx

```diff
@@ -1,11 +1,15 @@
-import { Flex, LoadingIndicator } from '@webkom/lego-bricks';
+import { Flex, Icon, LoadingIndicator } from '@webkom/lego-bricks';
+import { ListFilter } from 'lucide-react';
+import moment from 'moment';
+import { useEffect, useState, type CSSProperties } from 'react';
 import CommentForm from 'app/components/CommentForm';
 import { generateTreeStructure } from 'app/utils';
+import Dropdown from '../Dropdown';
```

Comment:
Absolute import please 😄 I understand that this was probably just an auto import though